src/backend/distributed/utils/citus_readfuncs.c

```diff
@@ -45,9 +45,15 @@ CitusSetTag(Node *node, int tag)
 	nodeTypeName *local_node = (nodeTypeName *) CitusSetTag((Node *) node, T_##nodeTypeName)
 
 /* And a few guys need only the pg_strtok support fields */
+#if PG_VERSION_NUM >= 120000
 #define READ_TEMP_LOCALS()	\
-	char	   *token;		\
+	const char	*token;		\
```

Comment:
What happens if we use `const` also in pre-12?

Reply:
In 12 we get warnings that token is receiving `const`. Before 12 we get warnings that `const` is being discarded